<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="TreesTW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="AddValue">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="DeleteValue">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddManyButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Many</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddManyValue">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
